src/plugin.scss

```diff
@@ -155,6 +155,7 @@ $placeholder-background-color: #303030;
 
 // Horizontal playlist orientation
 .vjs-playlist-horizontal {
+  overflow-x: auto;
```

Comment:
we probably need to put the other dimension has `hidden` specifically or make the default be hidden since I'm still seeing a vertical scrollbar in the horizontal playlist

Reply:
Good point, done.